### ISSUE TITLE
Cannot compile

### ISSUE THREAD

**Issue by Irides-Chromium** [Legimet/nPDF#21]
- Cannot find "hb.h" and "hb-ft.h".
- Cannot find "gnu/stubs.h" and "gnu/stubs-32.h"
- After working around the two above, there are more errors, see in the log file for detail

[nPDF.log.txt](https://github.com/Legimet/nPDF/files/887167/nPDF.log.txt)

**Comment by adriweb** [comment]
Yeah the fact that it has included `/usr/include/stdio.h` is a bad sign...

**Comment by Irides-Chromium** [comment]
What do you mean? So it shouldn't include `/usr/include/stdio.h`?

**Comment by Legimet** [comment]
It looks like your SDK isn't setup correctly. The headers from your ARM toolchain are conflicting with your system's headers.

**Comment by Irides-Chromium** [comment]
So what should I do? I set up the SDK according to Ndless's README.

**Comment by Vogtinator** [comment]
I tried to build it, works fine for me. Note that you need to use ```make```, not ```make -j8``` or similiar as it has some issues.

**Comment by Irides-Chromium** [comment]
Yes, I used `make` without any arguments.
